An article about dns leaks (#87)
AdguardTeam/KnowledgeBaseVPN: docs/general/dns-leaks.md

---

> title: 'What is a DNS leak'

Reviewer: What is a DNS leak?

sidebar_position: 9
---

Once you install a VPN, you want to be sure that there won't be any leaks that will lead to your de-anonymization. AdGuard VPN reliably protects your traffic and your data from prying eyes. Nevertheless, in some cases you may suspect the presence of DNS leaks in our VPN service. We'll explain where these concerns come from and why they are unfounded.

> ## What are DNS leaks

Reviewer: What are DNS leaks?

> DNS leaks when using a VPN is a situation where a connection to a DNS server goes outside your encrypted VPN tunnel and your data becomes visible to the ISP.  

Reviewer: DNS leaks when using a VPN is a situation — звучит несогласованно

When you use a VPN, a connection to a DNS server may travel outside of the encrypted VPN tunnel and your traffic may become visible to the ISP.

> In other words, every time you open a website such as `google.com`, your browser sends a request to the ISP's DNS server and in return receives the IP address of the requested website. In this way, your ISP sees all your online activity and can track you.

Reviewer: such as `google.com` можно и убрать

## How to detect DNS leaks

> There are all sorts of security scanner websites for detecting DNS leaks, such as `whoer.net`. It is worth understanding that these sites themselves are not perfect and their algorithms are not clear, as opposed to their intentions to intimidate users with imaginary leaks and potentially sell some service.

Reviewer: security scanner websites —> anonymity check services

Reviewer: It is worth understanding —> It should be understood

> Some security scanner websites consider the coincidence of the user's IP address and the DNS server's IP address to be a "good" result, indicating that there is no leakage. In reality, this is not the case: such a match may indicate the use of a VPN. In the usual state, when there is no VPN connection and the requests go to your ISP's DNS, the IP address of the DNS server and your own do not coincide.

Reviewer: security scanning website

Reviewer: ```suggestion
Some security scanning websites consider the coincidence of the user's IP address and the DNS server's IP address to be a "good" result, indicating that there are no leaks. In reality, such a match may indicate the use of a VPN. When VPN is disabled and the requests go to your ISP's DNS server, the IP address of the DNS server and your own one do not coincide.
```

## Why there are no DNS leaks in AdGuard VPN 

> If you enable AdGuard VPN on your device and run a check on any of the scanner websites, it finds that DNS queries go to the [AdGuard DNS](https://adguard-dns.io/welcome.html) servers. However, this cannot be considered a leak.

Reviewer 2: ```suggestion
If you enable AdGuard VPN on your device and run a check on any of the scanner websites, it finds that DNS queries go to the [AdGuard DNS](https://adguard-dns.io/) servers. However, this cannot be considered a leak.
```

> If you enable AdGuard VPN on your device and run a check on any of the scanner websites, it finds that DNS queries go to the [AdGuard DNS](https://adguard-dns.io/en/welcome.html) servers. However, this cannot be considered a leak.

Reviewer: https://adguard-dns.io/welcome.html

> Using a non-filtering AdGuard DNS server is set by default in AdGuard VPN. This ensures that your queries do not go to the DNS servers of your ISP, which means that you keep your privacy and anonymity on the Internet. 

Reviewer: ```suggestion
Using a non-filtering AdGuard DNS server is set by default in AdGuard VPN. This ensures that your queries do not go to the DNS servers of your ISP, which means that you stay private and anonymous online.
```

> What's more, AdGuard DNS is a very popular DNS service, used by over 50 million people. What does this mean for you? Imagine the following situation: the usual VPN service uses the DNS server which is located at the same IP address as the VPN server. The number of its users hardly exceeds 1,000.

Reviewer: 50 million??

Reviewer: The number of its users hardly exceeds 1,000. — это Костя сказал, да?

Author: про 50 млн - инфа с сайта
про 1000 юзеров - инфа от Андрея

> And in the case of AdGuard DNS you will "merge" with 50 million users so nobody will be able to track you by the fact of using the DNS server.

Reviewer 2: ```suggestion
And in the case of AdGuard DNS you will "merge" with 50 million users so nobody will be able to track you by the mere fact of using that DNS server.
```

> And in the case of AdGuard DNS you will "merge" with 50 million users, and you can't be tracked just by the fact of using the DNS server.

Reviewer: ```suggestion
And in the case of AdGuard DNS you will "merge" with 50 million users so nobody will be able to track you by the fact of using the DNS server.
```
но вообще звучит так, будто у нас один сервер на 50 (35) миллионов пользователей

## How to set up a custom DNS server in AdGuard VPN

> There are many popular public DNS servers from [well-known DNS providers](https://adguard-dns.io/kb/en/general/dns-providers/). Some of them can only perform their direct duties – giving the IP addresses of the requested domains, and some can do more. 

Reviewer: https://adguard-dns.io/kb/general/dns-providers

> For example, the filtering AdGuard DNS removes ads and protects your device from being tracked, while AdGuard DNS Family Protection combines AdGuard DNS features with Safe search and Parental control.

Reviewer: the filtering AdGuard DNS — этоо название серера или что?

Author: Не, название у него просто AdGuard DNS. Ноо, почему-то захотелось сделать акцент на то, что он фильтрующий.

---

> title: 'What a DNS leak is'

Reviewer: не, тут лучше было оставить с вопросом: посмотри на оглавление днс кб: What is DNS? / What is AdGuard DNS? / What is Private AdGuard DNS?
